STATUS-ERROR-CODES.md

```diff
@@ -0,0 +1,109 @@
+# Summary of status- and error-codes to be returned by an implementation
+
+The follow HTTP status codes should be used by the ITSM REST implementation.
+They are recognized and can be dealt with by our client implementation:
+
+## Success
+
+### 200: Ok
+
+Successful GET or PUT requests.
+
+### 201: Created
+
+Successful POST requests (note that the new resource's URL needs to be
+contained in the "Location" header in the response).
+
+## Client errors
+
+Note that any '4**' error code indicates that it's possible for the client
+to fix the situation by changing something in their request. Thus, for any
+'4**' error, the return body should contain a human readable error message
+explaining the problem. The maintainers of the client code can see those
+messages and can use it to improve the cient implementation.
+
+In general, any requests that received a '4**' response will NOT be automatically
+retried by the client, but may be retried at a later time if and when manual fixes
+have been applied to the client code or configuration.
+
+### 400: Bad Request
+
+If there is something wrong with the request body (in case of POST or PUT),
+or if the query parameters (for some GET requests) are wrong. This includes
+situation where the request is correctly formatted (correct syntax), but
+where the request is still wrong in a sematic sense. An example here is the
+creation of a user/customer association, where the specified association
+exists already.
+
+### 401: Unauthorized
+
+The client could not be authenticated (note that the official name of this
+return code is "Unauthorized", but by convention, it is used to indicate
+that the authentication failed. This can happen if the client tried to access
```

Comment:
I think after the word `failed` is where you should put a closing bracket `)`

Reply:
done